ansible/vars/dev/dev_private.yml

```diff
@@ -0,0 +1,42 @@
+secrets:
+  ANSIBLE_USER_PASSWORD_SHA_512: 'YIN9OONyKqkxY'
```

Comment:
Would be useful to retain the comment containing the actual password here (`# blahblah`).